lib/src/main/java/vmath/Mat4.java

```diff
@@ -47,5 +47,19 @@ public float get(int i, int j) {
             throw new IndexOutOfBoundsException();
         }
         return m[i * 4 + j];
-    }
-}
+    } 
+
+    public Mat4 mul(Mat4 other) {
+        Mat4 result = new Mat4();
+        for (int i = 0; i < 4; i++) {
+            for (int j = 0; j < 4; j++) {
+                float value = 0.0f;
+                for (int k = 0; k < 4; k++) {
+                    value += this.get(i, k) * other.get(k, j);
+                }
+                result.m[i * 4 + j] = value;
+            }
```

Comment:
One last comment: our code style for this project states that there should be no spaces around * and / operators. Please remove these spaces when you have a chance.